src/pybind/mgr/rook/module.py

```diff
@@ -348,7 +348,24 @@ def describe_service(self,
                     size=active,
                     last_refresh=now,
                 )
-
+        if service_type == 'osd' or service_type is None:
+            # OSDs
+            all_osds = self.rook_cluster.get_osds()
+            for osd in all_osds:
+                svc = 'osd.' + osd.metadata.labels['osd']
+                if svc in spec:
+                    continue
+                spec[svc] = orchestrator.ServiceDescription(
+                    spec=DriveGroupSpec(
```

Comment:
An OSD service is supposed to group OSDs. I'd actually group all OSDs into a single dummy DriveGroupSpec.